qrify/qrify

```diff
@@ -83,6 +83,23 @@ makeqr()
   httpGet qrenco.de/${input} ## get a response for the qrcode
 }
 
+# redirects the image obtained from the goqr api into a png file
+function makeQRFile {
+	input=$(echo ${input} | sed s/" "/%20/g ) ##same as in the makeqr function
+	httpGet "api.qrserver.com/v1/create-qr-code/?size=150x150&data=${input}" > "$fileName"
+}
+
+# storing can only be done in a png file, so fail for all other cases
+function validateFileExt {
```

Comment:
This shouldn't need to be here, if there is only only one possible format the ```-f``` option should only let the user change the filename when you go to create the file just append .png to the filename given (prevent user error a little more often)